notebooks/en/enterprise_cookbook_overview.md

```diff
@@ -0,0 +1,52 @@
+# Enterprise Hub Cookbook
+
+The Enterprise Hub Cookbook is designed for power users and enterprises who want to go beyond the standard free features of the Hugging Face Hub and integrate machine learning deeper into their production workflows. The cookbook guides you through a selection of recipes (Jupyter Notebooks) with copy-pastable code to help you get started with the Hub's advanced features.
+
+<Youtube id="CPQGBn-yXJQ"/>
+
+
+## Interactive Development in HF Spaces \[coming soon\]
```

Comment:
I think the brackets are typically used when there are two identical headers, but we want to differentiate the two. Let's try using parentheses instead?

Reply:
ok, updated it to the simpler "(coming soon)"